packages/ui-img/src/Img/props.ts

```diff
@@ -40,6 +40,10 @@ type ImgOwnProps = {
   src: string
   alt?: string
   display?: 'inline-block' | 'block'
+  /**
+   * Gets passed down to  the img component. Same as the native HTML img's loading attribute
+   */
+  loading?: 'eager' | 'lazy'
```

Comment:
Please add it to the ignored props in the example generation, it renders a lot of unneeded VRT examples